prow/statusreconciler/controller_test.go

```diff
@@ -66,10 +71,30 @@ func TestAddedBlockingPresubmits(t *testing.T) {
 			name: "added non-reporting presubmit means no added blocking jobs",
 			old: `"org/repo":
 - name: old-job
-  context: old-context`,
+  context: old-context
+  always_run: true`,
+			new: `"org/repo":
+- name: old-job
+  context: old-context
+  always_run: true
+- name: new-job
+  context: new-context
+  always_run: true
+  skip_report: true`,
+			expected: map[string][]config.Presubmit{
+				"org/repo": {},
+			},
+		},
+		{
+			name: "added presubmit that needs a manual trigger means no added blocking jobs",
+			old: `"org/repo":
+- name: old-job
+  context: old-context
+  always_run: true`,
 			new: `"org/repo":
 - name: old-job
   context: old-context
+  always_run: true
 - name: new-job
   context: new-context
   skip_report: true`,
```

Comment:
This should be removed for experimental control. This test case may succeed because the new job doesn't report, but we really want to be checking that the test case succeeds because the new job is manually triggered.

Reply:
Err, should add `always_run` to it